src/Umbraco.Web.UI/Umbraco/config/lang/da.xml

```diff
@@ -647,6 +647,7 @@
     <key alias="design">Design</key>
     <key alias="dictionary">Ordbog</key>
     <key alias="dimensions">Dimensioner</key>
+    <key alias="discard">Discard</key>

```

Comment:
@nielslyngsoe "Kassér" would be a better for Danish translation 👍

Reply:
@bjarnef What a bummer, I fully agree. :-)